providers/apache/spark/pyproject.toml

```diff
@@ -77,6 +80,7 @@ dev = [
     "apache-airflow-devel-common",
     "apache-airflow-providers-cncf-kubernetes",
     "apache-airflow-providers-common-compat",
+    "apache-airflow-providers-openlineage",
     # Additional devel dependencies (do not remove this line and add extra development dependencies)
```

Comment:
@potiuk Pre-commit fails for me for this locally. Do I need to change any other file too?

Reply:
Just add it BELOW the comment :)